docs/en/latest/plugins/openfunction.md

```diff
@@ -0,0 +1,181 @@
+| function_uri                | string  | True     |         |              | function uri. For example, `https://localhost:30858/default/function-sample`.                              |
+| ssl_verify                  | boolean | False    | true    |              | When set to `true` verifies the SSL certificate.                                                           |
+| authorization               | object  | False    |         |              | Authorization credentials to access functions of OpenFunction.                                      |
+| authorization.service_token | string  | False    |         |              | The token format is 'xx:xx' which support basic auth for function entry points.                                      |
+| timeout                     | integer | False    | 3000ms  | [100, ...]ms | OpenFunction action and HTTP call timeout in ms.                                                              |
+| keepalive                   | boolean | False    | true    |              | When set to `true` keeps the connection alive for reuse.                                                   |
+| keepalive_timeout           | integer | False    | 60000ms | [1000,...]ms | Time is ms for connection to remain idle without closing.                                                  |
+| keepalive_pool              | integer | False    | 5       | [1,...]      | Maximum number of requests that can be sent on this connection before closing it.                          |
+
+:::note
+
+The `timeout` attribute sets the time taken by the OpenFunction to execute, and the timeout for the HTTP client in APISIX. OpenFunction calls may take time to pull the runtime image and start the container. So, if the value is set too small, it may cause a large number of requests to fail.
+
+:::
+
+## Prerequisites
+
+Before configuring the plugin, you need to have OpenFunction running.
+Installation of OpenFunction requires a certain version Kubernetes cluster.
+For details, please refer to [Installation](https://openfunction.dev/docs/getting-started/installation/).
+
+### Install OpenFunction via Helm Chart
+
+The example below shows OpenFunction installed in Helm:
+
+```shell
+# add the OpenFunction chart repository
+helm repo add openfunction https://openfunction.github.io/charts/
+helm repo update
+
+#install the OpenFunction chart
+kubectl create namespace openfunction
+helm install openfunction openfunction/openfunction -n openfunction
+```
+
+You can then verify if OpenFunction is ready:
+
+```shell
+kubectl get pods --namespace openfunction
+```
+
+### Create and Push a Function
+
+You can then create a function follow the [sample](https://github.com/OpenFunction/samples)
+
+When building a function, you’ll need to push your function container image to a container registry like Docker Hub or Quay.io. To do that you’ll need to generate a secret for your container registry first.
```

Comment:
"`suggestion
You'll need to push your function container image to a container registry like Docker Hub or Quay.io when building a function. To do that, you'll need to generate a secret for your container registry first.
```